Include/API/lib3mf_model.hpp

```diff
@@ -185,6 +185,15 @@ class CModel : public virtual IModel, public virtual CBase {
 
 	void RemoveCustomContentType (const std::string & sExtension);
 
+	void AddCustomNameSpace (const std::string & sNameSpace, const std::string & sLocale);
```

Comment:
How about deciding the namespace prefix internally in lib3mf? Similar to what is done for MetaData namespaces. All the namespaces can be collected in a set (rather than map) in NMR_Model class. And while writing the 3dmodel file, generate custom namespace prefixes based on the count - xmlns:customXMLNS0, xmlns:customXMLNS1 and so on.